buffer/src/test/java/io/netty/buffer/AbstractByteBufTest.java

```diff
@@ -1895,11 +1968,11 @@ public void run() {
                         byte[] array = new byte[8];
                         buf.readBytes(array);
 
-                        Assert.assertArrayEquals(bytes, array);
+                        assertArrayEquals(bytes, array);
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Remove this assertion. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS2186)